src/sentry/scim/endpoints/schemas.py

```diff
@@ -0,0 +1,192 @@
+                    "mutability": "readWrite",
+                    "returned": "default",
+                    "uniqueness": "none",
+                },
+                {
+                    "name": "givenName",
+                    "type": "string",
+                    "multiValued": False,
+                    "description": "The given name of the User, Sentry does not support this attribute and will return N/A as a string for compatability purposes.",
+                    "required": False,
+                    "caseExact": False,
+                    "mutability": "readWrite",
+                    "returned": "default",
+                    "uniqueness": "none",
+                },
+            ],
+        },
+    ],
+    "meta": {
+        "resourceType": "Schema",
+        "location": "/v2/Schemas/urn:ietf:params:scim:schemas:core:2.0:User",
+    },
+}
+
+SCIM_GROUP_ATTRIBUTES_SCHEMA = {
+    "id": SCIM_SCHEMA_GROUP,
+    "name": "Group",
+    "description": "SCIM Group maps to Sentry Team",
+    "attributes": [
+        {
+            "name": "displayName",
+            "type": "string",
+            "multiValued": False,
+            "description": "A human-readable name for the Group. REQUIRED.",
+            "required": False,
+            "caseExact": False,
+            "mutability": "readWrite",
+            "returned": "default",
+            "uniqueness": "server",
+        },
+        {
+            "name": "members",
+            "type": "complex",
+            "multiValued": True,
+            "description": "A list of members of the Group.",
+            "required": False,
+            "subAttributes": [
+                {
+                    "name": "value",
+                    "type": "string",
+                    "multiValued": False,
+                    "description": "Identifier of the member of this Group.",
+                    "required": False,
+                    "caseExact": False,
+                    "mutability": "immutable",
+                    "returned": "default",
+                    "uniqueness": "none",
+                },
+                {
+                    "name": "$ref",
+                    "type": "reference",
+                    "referenceTypes": ["User"],
+                    "multiValued": False,
+                    "description": "The URI of the corresponding 'User' resource to which the user belongs.",
+                    "required": False,
+                    "caseExact": False,
+                    "mutability": "readOnly",
+                    "returned": "default",
+                    "uniqueness": "none",
+                },
+            ],
+            "mutability": "readWrite",
+            "returned": "default",
+        },
+    ],
+    "meta": {
+        "resourceType": "Schema",
+        "location": "/v2/Schemas/urn:ietf:params:scim:schemas:core:2.0:Group",
+    },
+}
+
+SCIM_SCHEMA_LIST = [SCIM_USER_ATTRIBUTES_SCHEMA, SCIM_GROUP_ATTRIBUTES_SCHEMA]
```

Comment:
I'm assuming that this 2 schemas are copied from somewhere so I didn't look too deeply into them.